### DISCUSSION TITLE
Builder login problems

### DISCUSSION THREAD

**Discussion by andyburgessmd** [Budibase/budibase#8578] [General]
Is anyone else experiencing any issues getting into builder today? Mine keeps looping back to the login page, but shows the 'logged in successfully' notification. 
I can access the application, just not the builder.

**Comment by andyburgessmd** [comment]
I should add this is on the cloud version.

**Comment by shogunpurple** [comment]
Hi @andyburgessmd - I'm not able to recreate this on our tenant, but noticed another user also reported it in discord. Is this still happening to you?

**Reply by andyburgessmd** [reply]
I think the user on discord is probably me as well.
It is still happening, Are you able to check on my tenant (mistraldata)? I have just noticed that in the apps I am seeing a non-logged in view and also the 'made with budibase' button in the bottom right is showing - we are on a paid plan.

**Reply by andyburgessmd** [reply]
@shogunpurple are you able to help or is there a support process that can help restore access?

**Reply by shogunpurple** [reply]
Hi @andyburgessmd - apologies I was in a meeting there. Yes I can help. 

Have you tried clearing your browser cookies or accessing via a n incognito tab? Does this make any difference?

Will check your tenant now too.

**Reply by andyburgessmd** [reply]
@shogunpurple yes I have tried both those and it made no difference. A colleague has also been able to replicate.

**Reply by shogunpurple** [reply]
@andyburgessmd is there any errors in the browser console whenever it happens, to help us debug? Everything on the tenant is looking ok and also in stripe.

**Reply by andyburgessmd** [reply]
![image](https://user-images.githubusercontent.com/110813989/200614083-7a3d8b8e-ab51-422a-bb46-ed4385c93603.png)
Yes, these show in console on https://mistraldata.budibase.app/builder/auth/login

**Reply by shogunpurple** [reply]
Thank you @andyburgessmd - ok, we think we see what may be wrong. It appears related to user groups - we will continue debugging and try to get a fix pushed out asap

**Reply by shogunpurple** [reply]
@andyburgessmd we have pushed a fix that we believe will address the problem. Can you try again when you are next online and let us know if the issue has been resolved? Thank you for reporting.

**Reply by andyburgessmd** [reply]
@shogunpurple thanks, I have tested this morning and I can access the builder and apps again. 
I have noticed odd things happening with the access levels when I use groups and there is already a user assigned to the application. Has I done something wrong?

**Reply by shogunpurple** [reply]
Hi @andyburgessmd - glad to hear that. What kind of odd things are you seeing happening?

**Reply by andyburgessmd** [reply]
@shogunpurple for one of my apps which is still in development I had one user (me) with admin access setting but when I tried to access the app (which had basic access set on all pages) I got a blank screen. When I then added a group with just myself in it and admin access again it started working and the app pages did show what I was expecting. 
Adding this group to the app is about the time that this main issue thread occured.
